settings.gradle.kts

```diff
@@ -0,0 +1 @@
+rootProject.name = "SLIV-tool"

```

Comment:
let's put all test data into separate folder

Reply:
it is not clear what is test data. Images?

Reply:
yes